Makefile

```
@@ -102,6 +102,10 @@ test-contracts:
 	done
 	@cd contracts/autonity/contract/ && $(NPMBIN)/truffle test && cd -
 
+docker-e2e-test: embed-autonity-contract
+	build/env.sh go run build/ci.go install
```

If your intent here is to build the autonity binary you can just depend on `autonity` like this, autonity depends on `embed-autonity-contract` so that will also be run.

```suggestion
docker-e2e-test: autonity
```